zbus/src/abstractions/mod.rs

```diff
@@ -9,6 +9,6 @@ pub(crate) mod async_lock;
 pub use async_drop::*;
 pub(crate) mod file;
 
-// Not macOS-specific itself but only used on macOS.
-#[cfg(target_os = "macos")]
+// Not unix-specific itself but only used on unix.
+#[cfg(target_family = "unix")]
```

Comment:
Could we please split this in a separate commit that comes before use of `process` on unix target?